website/addons/base/views.py

```diff
@@ -384,6 +384,31 @@ def create_waterbutler_log(payload, **kwargs):
     return {'status': 'success'}
 
 
+@file_signals.file_updated.connect
+def addon_delete_file_node(self, node, event_type, payload, user=None):
+    if event_type == 'file_removed' and payload.get('provider', None) != 'osfstorage':
```

Comment:
Can you add a docstring explaining why it's important to trash these instead of just deleting them?